xs2a-adapter-service-api-adapter/src/main/java/de/adorsys/xs2a/adapter/adapter/BasePaymentInitiationService.java

```diff
@@ -253,7 +253,42 @@ public BasePaymentInitiationService(Aspsp aspsp,
                                                                      String authorisationId,
                                                                      RequestHeaders requestHeaders,
                                                                      RequestParams requestParams) {
-        throw new UnsupportedOperationException();
+        return getPaymentInitiationScaStatus(paymentService,
```

Comment:
does it present in the remote client?

Reply:
yes

Reply:
assuming you mean `getPaymentInitiationScaStatus` and `getPaymentInitiationAuthorisation` methods